### ISSUE TITLE
RemovedInSphinx40Warning: force_decode() is deprecated.

### ISSUE THREAD

**Issue by jamadden** [repoze/repoze.sphinx.autointerface#13]
With Sphinx 2.0, I get these warnings:
```
repoze/sphinx/autointerface.py:99: RemovedInSphinx40Warning: force_decode() is deprecated.
  docstrings = [prepare_docstring(force_decode(doc, None))]
```

**Comment by freddrake** [comment]
I think **force_decode** is only interesting on Python 2, so this shouldn't be difficult to deal with.

**Comment by freddrake** [comment]
See PR https://github.com/repoze/repoze.sphinx.autointerface/pull/14.

**Comment by freddrake** [comment]
PR #11 takes a different approach to the other updates made in #14, but doesn't address the **force_decode** change.

**Comment by dataflake** [comment]
Fixed by #17 , version 1.0.0 is now published on PyPI.
